layouts/partials/hooks/body-end.html

```diff
@@ -112,16 +144,18 @@
 
         // If you added more searchable fields to the search index, list them here.
         this.field("title");
+        this.field("description")
         this.field("youtube");
         this.field("url");
+        this.field("thumbnail");
         this.field("type");
         this.field("year");
         this.field("date");
 
         for (var doc of request.response)
         {
           this.add(doc);
-          lookup[doc.title] = doc;
+          lookupp[doc.title] = doc;
```

Comment:
@thisisobate I was going to ask you about this lookupp because I thought it was an error, but it looks like it was declared this way on purpose... I guess you understand this part of the code and I do not, can you shed some light?